src/pages/Subscribe/Subscribe.js

```diff
@@ -1,7 +1,362 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { useNavigate, useLocation } from 'react-router-dom';
 
 const Subscribe = () => {
-  return <h1>Subscribe</h1>;
+  const { pathname } = useLocation();
+
+  const [index, setIndex] = useState(1);
+  const [name, setName] = useState('');
+  const [address, setAddress] = useState('');
+  const PRICE = 39000;
+
+  const result = PRICE * index;
+
+  const navigete = useNavigate();
+
+  const goToBack = () => {
+    navigete(-1);
+  };
+
+  const onChangeName = e => {
+    setName(e.target.value);
+  };
+  const onChangeAddress = e => {
+    setAddress(e.target.value);
+  };
+
+  const onIndex = e => {
+    setIndex(e.target.value);
+  };
+
+  const onClickNaverBtn = e => {
+    alert('네이버페이로 결제하시겠습니까?');
+  };
+
+  const onClickKakaoBtn = e => {
+    alert('카카오페이로 결제하시겠습니까?');
+  };
+
+  const onClickCardBtn = e => {
+    alert('신용카드로 결제하시겠습니까?');
+  };
+
+  useEffect(() => {
+    window.scroll(0, 0);
+  }, [pathname]);
+
+  let token = localStorage.getItem('token') || '';
+  return (
+    <div>
+      <Backbutton onClick={goToBack}>뒤로가기</Backbutton>
+      <SubscribeTitle>매월 몇 개의 담아박스가 필요하세요?</SubscribeTitle>
+      <SubscribeTitleBox>
+        <SubscribeTitleImage
+          src="https://cdn-icons-png.flaticon.com/512/679/679720.png"
+          alt="담아박스 이미지"
+        />
+        <RigntTextWapper>
+          <p>담아박스</p>
+          <SelectBox>
+            <SelectBoxIcon>x</SelectBoxIcon>
+            <SelectBoxList value={index} onChange={onIndex}>
```

Comment:
```suggestion
            <SelectBoxList value={userInfo.index} onChange={getUserInfo}>
```
state가 객체로 바뀌면 이렇게 수정 해주세요